spec/jobs/moab_replication_audit_job_spec.rb

```diff
@@ -45,12 +45,11 @@
 
     it 'calls PartReplicationAuditJob once per related endpoint' do
```

Comment:
i'd probably drop "related" from the test description

Reply:
Done.